lib/elixir/pages/Guards.md

```diff
@@ -173,15 +179,18 @@ defmodule Check do
   def empty?(_val), do: false
 end
 
-Check.empty?(%{}) #=> true
-Check.empty?({}) #=> true
+Check.empty?(%{})
+#=> true
+
+Check.empty?({})
+#=> true
 ```
 
 ## Defining custom guard expressions
 
-As mentioned before, only the expressions listed in this page are allowed in guards. However, we can take advantage of macros to write custom guards that can simplify our programs or make them more domain-specific. At the end of the day, what matters is that the *output* of the macros (which is what will be compiled) boils down to a combinations of the allowed expressions.
+As mentioned before, only the functions and operators listed in this page are allowed in guards. However, we can take advantage of macros to write custom guards that can simplify our programs or make them more domain-specific. At the end of the day, what matters is that the *output* of the macros is boiled down to a combination of allowed expressions.
 
-Let's look at a quick case study: we want to check that a function argument is an even or odd integer. With pattern matching, this is impossible to do since there are infinite integers, and thus we can't pattern match on the single even/odd numbers. Let's focus on checking for even numbers since checking for odd ones is almost identical.
+Let's look at a quick case study: we want to check whether a function argument is an even or an odd integer. With pattern matching this is impossible because there is an infinite number of integers, and therefore we can't pattern match on every single one of them. So, let's focus on checking for even numbers since checking for the odd ones is almost identical.
```

Comment:
"So, " is not necessary.

Reply:
removed the "So". Please have a look again, I'm trying to say "Let's do this" twice in the same paragraph.